internal/pkg/template/templates/workloads/partials/cf/sidecars.yml

```diff
@@ -47,6 +47,13 @@
 {{include "image-overrides" . | indent 2}}  
 {{- if $sidecar.Port}}
   PortMappings:
+{{- if $.ServiceConnect }} # remove when release
+{{- if eq $.HTTPTargetContainer.Container $sidecar.Name}}
+{{- if eq $.HTTPTargetContainer.Port $sidecar.Port}}
+      Name: TargetPort
```

Comment:
shouldn't the array mark be here? `-`? or just move this to the bottom of the object?

Reply:
i was just dumb. Good find!